<!-- This is an example launch file for how a ROS application could use a node -->
<launch>
   
	<arg name="use_face_detect" default="true"/>
	<arg name="test" default="false"/>
	<arg name="test_input" default="10"/>
	<arg name="test_id" default="default"/>
  	<group if="$(arg use_face_detect)">
	  		<rosparam file="$(find harmoni_face_detect)/config/configuration.yaml"/>
	  		<param name="test" value="$(arg test)"/>
			<param name="test_input" value="$(arg test_input)"/>
			<param name="test_id" value="$(arg test_id)"/>
			<node pkg="harmoni_face_detect" type="detector.py" name="face_detect" output="screen"/>
  	</group>


</launch>
